src/13-ref2vec/01-artists.ts

```diff
@@ -0,0 +1,149 @@
+import { ARTIST_IDS } from "./ids/artist-ids"
+import weaviate, { generateUuid5 } from "weaviate-ts-client"
+import _ from "lodash"
+import { Artist } from "./types/types"
+import { metaphysics } from "system/metaphysics"
+
+// Constants
+const CLASS_NAME: string = "SmallNewTrendingArtists"
+const BATCH_SIZE: number = 100
+
+const client = weaviate.client({
+  scheme: "https",
+  host: "https://weaviate.stg.artsy.systems",
+})
+
+async function main() {
+  const artists = await fetchArtists()
+
+  console.log(`Fetched ${artists.length} artists`)
+
+  await prepareCollection()
+  await insertObjects(artists, BATCH_SIZE)
+}
+
+main()
+
+async function prepareCollection() {
+  const client = weaviate.client({
+    scheme: "https",
+    host: "https://weaviate.stg.artsy.systems",
+  })
+
+  const alreadyExists = await client.schema.exists(CLASS_NAME)
+
+  if (alreadyExists) {
+    console.log(`${CLASS_NAME} class already exists, deleting it`)
+    await client.schema.classDeleter().withClassName(CLASS_NAME).do()
+  }
+
+  const classWithProps = {
+    class: CLASS_NAME,
+    moduleConfig: {
+      "text2vec-openai": {
+        model: "text-embedding-3-small",
+        dimensions: 1536,
+        type: "text",
+        vectorizeClassName: false,
+      },
+    },
+    vectorizer: "text2vec-openai",
+    properties: [
+      {
+        name: "name",
```

Comment:
**thought (non-blocking):** Interesting, my instinct would be to skip vectorization of a person's name, but I wonder now. 

My assumption is that names are sort of arbitrary and don't have much semantic content per se. But then again the embedding models may have learned associations between different names (say, artists of the same school — but that would only apply to names in the training data). 

Seems like something worth settling, but not a blocker for this PR.

Reply:
>Seems like something worth settling, but not a blocker for this PR.

Good points, and I agree. My inclination is to work through what field names to vectorize as we experiment with optimizing the results.